crates/bevy_pbr/src/atmosphere/types.wgsl

```diff
@@ -7,6 +7,9 @@ struct Atmosphere {
     // Radius at which we consider the atmosphere to 'end' for out calculations (from center of planet)
     top_radius: f32, // units: m
 
+    // The origin of the view relative to the center of the planet.
```

Comment:
Oh i see, its the origin of the view not of the planet. I'd invert this.

Reply:
You bring up a good point about the `atmosphere.origin` property referring to the planet's origin. I think having a negative value and then subtracting it from the current view position is less intuitive since the setting is moving observer relative to the earth rather than the earth itself. Maybe it's better to call this `view_origin` then?

Reply:
Can we just have the Atmosphere be a Component on an Entity with a Transform and then read the GlobalTransform property so that it plays well with big_space?

Reply:
Fair, but then we will need to start thinking about culling and multiple atmospheres support right? We could treat it as a sphere for culling. And for multiple atmosphere entities, reusing the luts for optimization in case the parameters are the same.

Maybe it's best if I revert this change for adding the origin in this PR, keep the offset to exactly the bottom radius as before, and work on this as a separate PR.

Reply:
Moved conclusion from here to #20770